sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -11,8 +11,8 @@
   "paths": {
     "/configurations/{id}": {
       "get": {
-        "summary": "Retrieve a configuration for Iot Hub devices and modules by it identifier.",
-        "description": " For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "summary": "Retrieve a configuration for Iot Hub device or module.",
```

Comment:
Shouldn't it be better to reword this as "Retrieve device or module configuration"?

Reply:
Don't have strong opinions either way. I can reword to your suggestion.

Reply:
since it is a get on a configuration, I like Sindhu's current wording. The configuration sits above any individual device or module, so starting the sentence with "Retrieve a device or module..." would be confusing